reducers/gotPlayer/peers.js

```diff
@@ -0,0 +1,99 @@
+import { playerPeersActions } from '../../actions';
+import { SORT_ENUM } from '../utility';
+import { combineReducers } from 'redux';
+
+const initialState = {
+  loading: true,
+  error: false,
+  loaded: false,
+  peerList: [],
+  sortState: '',
+  sortField: '',
+  sortFn: f => f,
+};
+
+const peers = (state = initialState, action) => {
+  switch (action.type) {
+    case playerPeersActions.REQUEST:
+      return {
+        ...state,
+        loaded: false,
+        loading: true,
+      };
+    case playerPeersActions.OK:
+      return {
+        ...state,
+        loading: false,
+        loaded: true,
+        peerList: [...action.payload],
+      };
+    case playerPeersActions.ERROR:
+      return {
+        ...state,
+        loading: false,
+        loaded: false,
+        error: true,
+      };
+    case playerPeersActions.SORT:
+      return {
+        ...state,
+        sortState: action.sortField === state.sortField ? SORT_ENUM.next(SORT_ENUM[state.sortState]) : SORT_ENUM[0],
+        sortField: action.sortField,
+        sortFn: action.sortFn,
+      };
+    default:
+      return state;
+  }
+};
+
+
+const byId = (state = {}, action) => {
```

Comment:
do we have to set this up for every endpoint?  It feels very verbose, can we refactor it so we only do it once?